crates/uv-interpreter/src/virtualenv.rs

```diff
@@ -73,13 +73,14 @@ pub(crate) fn virtualenv_from_env() -> Option<PathBuf> {
 
 /// Locate a virtual environment by searching the file system.
 ///
-/// Finds a `.venv` directory in the current or any parent directory.
+/// Checks if the working directory is a virtual environment.
+///
+/// If not, finds a `.venv` directory in the current or any parent directory.
 pub(crate) fn virtualenv_from_working_dir() -> Result<Option<PathBuf>, Error> {
```

Comment:
Thanks for merging !

The comment is not in line with the strategy.

Should I make a PR or let you fix it ?

Reply:
https://github.com/astral-sh/uv/pull/3406